caps/templates/caps/search_results.html

```diff
@@ -36,11 +31,27 @@ <h1 class="mb-3">Search inside plans</h1>
                     <div class="form-group">
                         <label for="council_name" class="d-flex flex-wrap flex-row-reverse">
                             <span class="text-muted">(Optional)</span>
-                            <span class="mr-auto">Council name contains</span>
+                            <span class="mr-auto">Council name</span>
                         </label>
                         <input type="text" class="form-control" name="council_name" id="council_name" value="{{ form.council_name.value|default_if_none:'' }}">
                     </div>
                   {% endif %}
+
+                    <p>Search method</p>
```

Comment:
FYI radio buttons should be grouped inside a `<fieldset>`, eg:

```html
<fieldset class="form-group">
    <legend style="font-size: 1em; font-weight: bold;">Search method</legend>
    <div class="form-check">
        <input … >
        <label … >
    </div>
    …
</fieldset>
```

Sorry it looks like we have pretty obnoxious default styling on `<legend>` elements, hence the inline styles in the code above. We should probably just fix the default. (Upgrading the site to Bootstrap 5 would help with this, and is also been something I’ve been meaning to do for ages.)

Reply:
Made this change.